plugins/destination/meilisearch/docs/overview.md

```diff
@@ -61,20 +61,19 @@ This is the spec used by the Meilisearch destination plugin.
 
   PEM-encoded certificate authorities.
   When set, a certificate pool will be created by appending the certificates to the system pool.
-  See [file variable substitution](/docs/advanced-topics/environment-variable-substitution#file-variable-substitution-example)
-  for how to read this value from a file.
+  See [file variable substitution](/docs/advanced-topics/environment-variable-substitution#file-variable-substitution-example) for how to read this value from a file.
 
 - `batch_size` (`integer`) (optional) (default: `1000`)
 
-  This parameter controls the maximum amount of items may be grouped together to be written as a single write.
+  Maximum amount of items that may be grouped together to be written in a single write.
 
 - `batch_size_bytes` (`integer`) (optional) (default: `4194304` (= 4 MiB))
 
-  This parameter controls the maximum size of items that may be grouped together to be written as a single write.
+  Maximum size of items that may be grouped together to be written in a single write.
 
 - `batch_timeout` (`duration`) (optional) (default: `20s`)
 
-  This parameter controls the timeout for writing a single batch.
+  Timeout for writing a single batch.
```

Comment:
actually it's the max interval between batches & should reflect so (the doc is wrong here)

Reply:
Out of scope, I will update all `batch_timeout` parameter docs in a separate PR?